### ISSUE TITLE
Wont connect to adafruit io

### ISSUE THREAD

**Issue by clebarron** [openhomeautomation/adafruit-io-dashboard#4]
Now it compiles and everything runs great but returns a dropped connection, the wifi connects sucessfully and is online but wont connect to adafruit io code uncchanged on my end.. thanks
aio connect failed..

**Comment by marcoschwartz** [comment]
Do any other Adafruit IO sketched work ?

**Comment by clebarron** [comment]
Yes I can upload and pass on the cc3000test sketch it connects, when i compile yours it runs through with only a momentary error about WLAN_SSID WLAN_PASS it shows a ^ below the W says max 32 but it compiles past it when uploaded it runs fine through the wifi connection get and displayes ip etc then goes strait to droppped connection, im not coding guru, seems like its still pubsub client issue.

**Comment by clebarron** [comment]
also I managed to piece together a sketch replacing Pubsub with the client from adafruit io arduino from over at the adfruit git, it is currently working flawless and uploading to the io, but because I dont quite know what im doing lol I didnt save the sketch so its running but I dont have a copy.. going to re write soon, at least from my not a coder mind its in the Pubsubclient somewhere..

**Comment by dgcarlucci** [comment]
getting the same thing, clebarron did you ever post a sketch fix somewhere using adafruit io instead of PubSub?

**Comment by dgcarlucci** [comment]
clebarron, did you ever post a solution somewhere using the adafruit io instead of Pubsub? i am getting the same issue....

**Comment by clebarron** [comment]
No I didnt post, unfortunately once i uploaded a working sketch I got so
excited i closed the sketch without saving. but  I started by getting
ADAFRUIT_MQTT Library, in it are examples of mqtt with the cc3000 wifi
connection, I started there as it has an out put of a light sensor, I
changed the code to fit my dht22 streams changing one line then duplicating
and finally got it all to work. but the home automation sketch I ended up
ditching and just rebuilding what I needed for my setup.. the connection is
pretty easy with adafruit mqtt, I am not savy enough to add mqtt and remove
pubsub from this git. sketch here.

On Thu, Mar 31, 2016 at 6:03 PM, dgcarlucci notifications@github.com
wrote:

> clebarron, did you ever post a solution somewhere using the adafruit io
> instead of Pubsub? i am getting the same issue....
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/openhomeautomation/adafruit-io-dashboard/issues/4#issuecomment-204195892

## 

Chris LeBarron

**Comment by dgcarlucci** [comment]
alright, i'll fiddle around, thanks! i saw a similar issue with the MQTT library from Adafruit here: https://github.com/adafruit/Adafruit_MQTT_Library/issues/31 could be a similar issue here as I was able to connect only after this fix.
